spec/functional/payment_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'active_record'
```

Comment:
Require `spec_helper` instead of `active_record`. Require for `active_record` should be use in `spec_helper.rb`

Reply:
It here looks like it doesn't need to require `spec_helper` , so I will just move `require 'active_record'` to spec_helper.rb.

Reply:
token_spec.rb also same.

Reply:
Yes, I've removed all the place where requires it in spec files.